test/test_slave.html

```diff
@@ -18,7 +18,13 @@
   <head>
     <title>PDF.js test slave</title>
     <meta charset="utf-8">
-    <script src="../build/generic/build/pdf.js"></script>
+    <script>
+      if (location.search.replace('?', '&').includes('coverage=true')) {
+        document.write('<script src="../coverage/build/generic/build/pdf.js"></script>');
+    } else {
+    document.write('<script src="../build/generic/build/pdf.js"></script>');
```

Comment:
The indentation in this whole code block is inconsistent. Please reformat the code to make the indentation consistent.